lib/swaps/Swaps.ts

```diff
@@ -140,7 +140,7 @@ class Swaps extends EventEmitter {
       this.sanitySwaps.set(rHash, sanitySwap);
       const swapClient = this.swapClientManager.get(currency)!;
       try {
-        await swapClient.addInvoice(rHash, 1);
+        await swapClient.addInvoice(rHash, 1, swapClient.cltvDelta);
```

Comment:
This should be handled in the same way. The first leg should include the second leg.

Reply:
In the case of `SanitySwap` and `SanitySwapAck` we're currently not exchanging CLTV delta information.

If it's OK with you I'd like to handle everything related to the sanity swaps in a separate issue/PR.

Reply:
I also think it's better to handle sanity swaps in a separate PR.